python/cudf/cudf/io/parquet.py

```diff
@@ -195,7 +196,12 @@ def to_parquet(
                 )
 
         return libparquet.write_parquet(
```

Comment:
This will return the metadata when `metadata_file_path` is set, but the `DataFrame` `to_parquet` method is not currently returning anything - So, [that method](https://github.com/rapidsai/cudf/blob/74bf304dcfb5c7f17019c73220d54a655b4b0b46/python/cudf/cudf/core/dataframe.py#L3799) will also need to add a `return` statement.

Reply:
thank you! fixed this on my end.